integration-cli/docker_api_swarm_test.go

```diff
@@ -1031,3 +1031,35 @@ func (s *DockerSwarmSuite) TestAPINetworkInspectWithScope(c *check.C) {
 	c.Assert(err, checker.IsNil, check.Commentf(string(out)))
 	c.Assert(status, checker.Equals, http.StatusNotFound, check.Commentf(string(out)))
 }
+
+func (s *DockerSwarmSuite) TestSwarmLBSandbox(c *check.C) {
+        d := s.AddDaemon(c, true, true)
+
+        // Create a new overlay network
+        out, err := d.Cmd("network", "create", "-d", "overlay", "new-overlay")
```

Comment:
The API suite should not use `d.Cmd()` please use `testEnv.APIClient()` to get a client, and make calls to the API.

Reply:
Thanks for the clarification.  That is what I thought, but, I saw other tests in the .Cmd() and hence the confusion.  I will clean this up.